app/helpers/table_helper.rb

```diff
@@ -0,0 +1,42 @@
+module TableHelper
+  def sort_table_header(heading, content_item_attribute, order, sort)
+    # Determine if this column is currently sorted on
```

Comment:
I have removed the comments and split the methods as suggested - also addressed the naming while pairing with @chao-xian. Looked into using a `class` but it started to get quite complicated giving the class access to the `ActionView` helpers such as `content_tag`, `params[]` especially in the tests and as such might not be worth it - more than willing to pair with you if you have a vision for how it could work.